test/controllers/categories_controller_test.rb

```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
+class CategoriesControllerTest < ActionDispatch::IntegrationTest
+  # test "the truth" do
```

Comment:
Given that there aren't any tests, make sure you run the application in development and can actually create a category from the app's web UI.